docker/kuberay-autoscaler/autoscaling_config.py

```diff
@@ -0,0 +1,282 @@
+from contextlib import suppress
+import logging
+import math
+import os
+from typing import Dict, Optional
+
+import json
+
+from ray.autoscaler._private.providers import _get_node_provider
+from ray.autoscaler._private.util import validate_config
+
+logger = logging.getLogger(__name__)
+
+# The name of this Ray cluster. Should coincide with the `metadata.name` field of the
+# Ray CR.
+_RAY_CLUSTER_NAME = os.getenv("RAY_CLUSTER_NAME")
+
+# The Kubernetes namespace in which this Ray cluster runs.
+_RAY_CLUSTER_NAMESPACE = os.getenv("RAY_CLUSTER_NAMESPACE")
+
+# Logical group name for the KubeRay head group.
+# Used as the name of the "head node type" by the autoscaler.
+_HEAD_GROUP_NAME = "head-group"
+
+_GPU_WARNING_LOGGED = False
+
+
+def generate_autoscaling_config() -> dict:
```

Comment:
```suggestion
def generate_autoscaling_config() -> Dict[str, Any]:
```

Reply:
Oh yeah, I always forget that dicts can have keys other than strings.